QubitServer/src/main/java/org/labrad/qubits/channeldata/IqDataBase.scala

```diff
@@ -0,0 +1,33 @@
+package org.labrad.qubits.channeldata
+
+import org.labrad.qubits.channels.IqChannel
+
+abstract class IqDataBase extends IqData {
+
+  private var channel: IqChannel = _
```

Comment:
OK, i was just confused because elsewhere (e.g. IqDataTime) it is explicitly set to null. I think we should go with whatever is idiomatic Scala. :)